jdk10-slim/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM openjdk:10-jdk-slim
+
+CMD ["gradle"]
+
+ENV GRADLE_HOME /opt/gradle
+ENV GRADLE_VERSION 4.8
+
+RUN apt-get update && apt-get install -y --no-install-recommends wget unzip
+RUN rm -rf /var/lib/apt/lists/*
```

Comment:
let's combine this with `&&`, also for readability, let's put each package installed on it's own line.